templates/issue.sample

```diff
@@ -79,5 +80,14 @@ function openFlyout() {
       </score>
     </CVSS>
   </classification>
+  
+  <knownvulnerabilities>
```

Comment:
I don't think this is going to work, "simple" tags are `<description>` or `<remedy>` where the content we're extracting is right there on the tag.

I think `<knownvulnerabilities>` is different. It's a collection of items, could be empty, could have multiple entries. And it's not just a collection of strings that we could convert into a bullet list, it's structured data, each vuln has :title, :severity, :references, etc.

If you split this PR into two, we can merge the OWASP2017 classification right away, but I suspect there is more to figure out on `:knownvulnerabilities` than what's here.